nncf/telemetry/wrapper.py

```diff
@@ -32,5 +33,9 @@
     nncf_logger.debug("openvino_telemetry package not found. No telemetry will be sent.")
     NNCFTelemetry = None
 
-if os.getenv(NNCF_CI_ENV_VAR_NAME) or os.getenv(NNCF_DEV_ENV_VAR_NAME) or NNCFTelemetry is None:
+# Currently the easiest way to disable telemetry in tests. Will break telemetry if pytest is used in NNCF package code.
```

Comment:
How will it work in a developer environment when pytest is included to it?

Reply:
`sys.modules` contains everything that has been imported. User code doesn't usually do `import pytest` in production - even in NNCF examples there isn't a single `import pytest` line.